apps/web/src/components/Table/ResponsiveTable/Table.tsx

```diff
@@ -97,7 +97,7 @@ const Row = (props: RowProps) => (
   <tr
     role="row"
     data-h2-display="base(flex) l-tablet(table-row)"
-    data-h2-background-color="base:selectors[*:nth-child(even)](background.dark.50) base:dark:selectors[*:nth-child(even)](white.3) base:selectors[*:nth-child(odd)](foreground)"
+    data-h2-background-color="base:children[*:nth-child(even)](background.dark.50) base:dark:children[*:nth-child(even)](white.3) base:children[*:nth-child(odd)](foreground)"
```

Comment:
This doesn't render as it did before on the tables. The [chromatic changeset](https://www.chromatic.com/review?appId=61e099a1bb1465003a73d3ce&number=9793&type=linked&view=changes) on this PR also caught the differences.

## Screenshot

<img width="1357" alt="Screen Shot 2024-03-19 at 13 40 38" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/43965012-b4ff-4609-9b3c-056bcd291198">